spec/controllers/hq/users_controller_spec.rb

```diff
@@ -22,7 +22,7 @@ def foobar_action
 
   before :each do
     sign_in instance_double(AdminUser)
-    @user = build_stubbed :user
+    @user = build_stubbed :user, email: 'someone@example.com'
```

Comment:
No I can't think why I would have explicitly assigned the email attribute. Will rmove this.